### DISCUSSION TITLE
FastApi

### DISCUSSION THREAD

**Discussion by roysti10** [anirudhRowjee/atlas#11] [Ideas]
This is a suggestion for the backend part of the project to  switch from flask to [FastApi](https://github.com/tiangolo/fastapi)
If you research about it, you would find a lot of articles claiming it to be better than flask or django

Writing tests using pytest is much easier with FastApi too

https://fastapi.tiangolo.com/

**Comment by anirudhRowjee** [comment]
I'm looking over the documentation, and It's got a lot more out of the box as opposed to flask. Seems like a solid choice.

**Reply by roysti10** [reply]
Yup and if you are comfortable with flask or django
Switching to FASTAPI should be easy as well

**Reply by anirudhRowjee** [reply]
Pretty comfortable with Django, so it should be good
